required/latex-lab/latex-lab-block-tagging.dtx

```diff
@@ -1346,23 +1346,26 @@
 %    will help with the transion. But instead of using a
 %    \env{trivlist} we use a blockenv and some tagging for the title
 %    (as a Caption).
-%
+%    We do not want the \texttt{Strong} tagging from \cs{textbf} here,
+%    so we use \cs{bfseries} to set the font.
```

Comment:
yes it isn't anymore (the text was added before). But imho `\bfseries` is better nevertheless, in case some document adds tagging to `\textbf`. I changed the text a bit.

Reply:
yes \bfseries is better (in fact \textbf is probably wrong as the user input may have or wants \par inside the caption)